### PR TITLE
Conduit: add utilities and examples variants to conduit

### PR DESCRIPTION
**Pull request by white238**

### CODE REVIEW

File: var/spack/repos/builtin/packages/conduit/package.py [hunk 1]
```diff
@@ -375,6 +377,19 @@ def hostconfig(self):
             if flags:
                 cfg.write(cmake_cache_entry("BLT_EXE_LINKER_FLAGS", flags, description))
 
+        #######################
+        # Examples/Utilities
+        #######################
+        if "+examples" in spec:
+            cfg.write(cmake_cache_entry("ENABLE_EXAMPLES", "ON"))
+        else:
+            cfg.write(cmake_cache_entry("ENABLE_EXAMPLES", "OFF"))
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/conduit/package.py:386]: Looks good.  Given the way this looks, the following would be tighter:

```suggestion
        cfg.write(cmake_cache_entry("ENABLE_EXAMPLES", "ON" if "+examples" in spec else "OFF"))
```

Comment (cyrush) [reply var/spack/repos/builtin/packages/conduit/package.py:386]: I would prefer commit style b/c its easier for my brain to parse and understand when scanning vs embedding an if in a function call. Is that ok?

**Review by tldahlgren — APPROVED** [review]
LGTM.

**Comment by tldahlgren** [comment]
I enabled auto merge under the assumption you are fine with the changes @cyrush .

**Review by cyrush — APPROVED** [review]

**Comment by cyrush** [comment]
Looks good,thanks!

**Comment by tldahlgren** [comment]
Strange that concretization took over 6 hours for two stacks.

@spackbot run pipeline

**Comment by cyrush** [comment]
thanks @white238 and @tldahlgren

**Pull request merged**